eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/FailureMessage.scala

```diff
@@ -139,19 +140,27 @@ object FailureMessageCodecs {
     }, (_: FailureMessage).code)
   )
 
+  private def failureOnionPayload(payloadAndPadLength: Int): Codec[FailureMessage] = Codec(
+    // We create payloads of the requested size: by always using the same size we ensure error messages are indistinguishable.
```

Comment:
That comment should be on method `failureOnionCodec`, it's the one which has the hardcoded default length.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2441/commits/5dd46b5847e3d980190da2e9715e23825cf107c6